app/controllers/application_controller.rb

```diff
@@ -3,12 +3,14 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery with: :exception
 
+  before_action :set_active_service_announcements
```

Comment:
Probably not totally ideal to do this on every request, open to suggestions on this.

Reply:
Since you're rendering the service announcement in the layout, it doesn't really matter. Regardless, I don't know why you're doing it as a before_action instead of just:

```ruby
def active_service_announcements
   @yolo ||= ServiceAnnouncement.all
end
```